docusaurus.config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   title: "effx | wiki",
   tagline: "The journey starts here",
   url: "https://effx.com/help",
```

Comment:
This needs an update

Reply:
NVM this seems specific to Github Pages: https://v2.docusaurus.io/docs/deployment#docusaurusconfigjs-settings